src/proxy-ui-api/frontend/src/views/Diagnostics/Diagnostics.vue

```diff
@@ -615,6 +615,17 @@ export default Vue.extend({
           return 'error';
       }
     },
+
+    messageLogEncryptionTooltipIconType(enabled: boolean): string {
+      switch (enabled) {
+        case true:
+          return 'warning-icon';
+        case false:
+          return 'disabled';
+        default:
+          return 'warning-icon';
+      }
```

Comment:
FYI (this is not a must, just an opinion)
I would do
```js
return enabled === false
        ? 'disabled'
        : 'warning-icon'; // <- ALL other cases

// OR

      switch (enabled) {
        case false:
          return 'disabled';
        case true:
        default:
          return 'warning-icon';
      }
```

❗I think every time there are 2 return values then there should be 2 return points for them.